.github/workflows/octokit.yml

```
@@ -13,24 +13,28 @@ on:
 
 jobs:
   ci:
-    name: Ruby ${{ matrix.ruby }}
+    name: 'Ruby ${{ matrix.ruby }} with Faraday ${{ matrix.faraday }}'
     runs-on: ${{ matrix.os }}-latest
 
     strategy:
       fail-fast: false
       matrix:
         os: [ ubuntu ]
         ruby: [ 2.5, 2.6, 2.7, head ]
+        faraday: [ '~> 0.16.0', '~> 0.17.0', '>= 1.0' ]
```

[note] `~> 1.0` may be better?

```suggestion
        faraday: [ '~> 0.16.0', '~> 0.17.0', '~> 1.0' ]
```